contracts/executable/AxelarExecutable.sol

```diff
@@ -68,7 +68,7 @@ abstract contract AxelarGMPExecutable is IAxelarGMPExecutable {
      * @notice Returns the address of the AxelarGMPGateway contract.
```

Comment:
```suggestion
     * @notice Returns the address of the AxelarGateway contract.
```
do a grep for these terms across the repo

Reply:
updated.